chromium_src/chrome/browser/ui/webui/password_manager/password_manager_ui.cc

```diff
@@ -26,6 +28,25 @@ void BraveAddPasswordManagerResources(content::WebUIDataSource* source,
                           IDR_BRAVE_PASSWORD_MANAGER_LOGO);   \
   BraveAddPasswordManagerResources(source, profile);
 
-#include <chrome/browser/ui/webui/password_manager/password_manager_ui.cc>
+#undef IDS_PASSWORD_MANAGER_UI_EMPTY_STATE_SYNCING_USERS
+#define IDS_PASSWORD_MANAGER_UI_EMPTY_STATE_SYNCING_USERS \
+  IDS_BRAVE_PASSWORD_MANAGER_UI_EMPTY_STATE_SIGNEDOUT_USERS
```

Comment:
I know this is a fairly common strategy we already use, but am I correct in assuming that we won't get any kind of error if this string ID changes upstream? If so, is there some lightweight way we can test for that?